### ISSUE TITLE
Item highlight is not visible enough in pure dark mode

### ISSUE THREAD

**Issue by tioalexis** [TapiocaFox/Daijishou#545]
When using pure dark mode, the contrast between the black background and the item highlight is too low, making it difficult to see which item is being highlighted.

This is the raw screenshot that I took, which doesn't show the problem properly:
https://ibb.co/z85gnHH

This is the edited screenshot to more closely resemble what I see on my device:
https://ibb.co/PhmLfDC

An option to modify the highlight visibility (or maybe even change the color) would be great.

**Comment by EgoBizarro** [comment]
Yeah, I also noticed this on my phone. It's especially hard to see when I'm outside.
